examples/trials/ir_enas/config.yml

```diff
@@ -0,0 +1,27 @@
+authorName: Unknown
+experimentName: enas_macro
```

Comment:
shall we put this example to a personal repo?

Reply:
> shall we put this example to a personal repo?

I suggest so. Shall we create a PR to `countif/enas_nni` which contains the example?